src/components/profile/MemberProjectsContainer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ProjectCard from '../cards/ProjectCard';
+
+class MemberProjectsContainer extends Component {
+  render() {
+    const { memberData } = this.props;
+    const projects = memberData.projects;
+
+    const projectsList = projects.map(project => {
+      return (
+        <ProjectCard
+          key={project.id}
+          projectUrl={project.projectUrl}
+          projectImageUrl={project.defaultPictureUrl}
+          avatarUrl={memberData.profilePictureUrl}
+          ownerProfileUrl={'/members/' + memberData.username}
```

Comment:
After updating getMemberProfile.js you can update the `ownerProfileUrl` prop 

```suggestion
          ownerProfileUrl={`/members/${memberData.memberId}`}
```